kubernetes-client/src/com/goyeau/kubernetes/client/operation/Listable.scala

```diff
@@ -11,15 +11,11 @@ import org.http4s.Method._
 
 private[client] trait Listable[F[_], Resource] {
   protected def httpClient: Client[F]
-  implicit protected val F: Sync[F]
   protected def config: KubeConfig
   protected def resourceUri: Uri
   implicit protected def listDecoder: Decoder[Resource]
 
-  @deprecated("Use list() instead", "0.4.0")
-  def list: F[Resource] = list()
```

Comment:
Removing this one is a little too soon. I don't know if anyone is still depending on it (or transitively). That would be a breaking change.

Reply:
that is not the only breaking change here. Changing the signatures from case classes to non-case classes is by itself a breaking change

Reply:
True. Leave it and the next version will be `0.5.0` then.